packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx

```diff
@@ -44,8 +42,8 @@ export const ActiveMembersList = () => {
       onPageChange={n => memberships?.fetchPage?.(n)}
       itemCount={memberships?.count || 0}
       pageCount={memberships?.pageCount || 0}
-      itemsPerPage={membershipsParams.memberships.pageSize}
-      isLoading={memberships?.isLoading || loadingRoles}
+      itemsPerPage={pageSize}
+      isLoading={(memberships?.isLoading && !memberships?.data) || loadingRoles}
```

Comment:
This avoids displaying the loading indicator if there's already data preserved in the cache-store since we're using `keepPreviousData`

https://github.com/user-attachments/assets/9be39c3e-5424-4522-9f8a-980a987f648f

Without checking for the cached data:

https://github.com/user-attachments/assets/98d5b113-5161-4312-a05c-1acd828925e0